src/commonMain/kotlin/spaceEngineers/controller/NextLayerControllerWrapper.kt

```diff
@@ -0,0 +1,65 @@
+package spaceEngineers.controller
+
+import spaceEngineers.commands.*
+import spaceEngineers.model.SeObservation
+import spaceEngineers.model.allBlocks
+
+class NextLayerControllerWrapper(
```

Comment:
I think classes like this should have some brief comment / documentation: purpose, why the name, etc.

I understand this is a draft, on the other hands, it's a pain to add comments later. I could be very brief for now and expanded on demand / later when the code is more stable.